### PR TITLE
Move triggers and remaining operators/sensors into standard provider

### PR DESCRIPTION
**Pull request by gopidesupavan**
Moving core area triggers and remaining operators/sensors inside standard provider

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: airflow/models/dag.py [hunk 1]
```diff
@@ -73,7 +73,7 @@
 
 from airflow import settings, utils
 from airflow.api_internal.internal_api_call import internal_api_call
-from airflow.assets import Asset, AssetAlias, BaseAsset
```
Comment (romsharon98) [inline airflow/models/dag.py]: why we removed it here?

File: generated/provider_dependencies.json [hunk 2]
```diff
@@ -846,6 +846,7 @@
     "plugins": [],
     "cross-providers-deps": [
       "amazon",
+      "common.compat",
```
Comment (romsharon98) [inline generated/provider_dependencies.json:849]: why we need this change here?

File: airflow/models/dag.py (hunk 1 above)
Comment (gopidesupavan) [reply airflow/models/dag.py]: Yeah, it removed by mistake when moving and import resolution, will update. :)

File: generated/provider_dependencies.json (hunk 2 above)
Comment (gopidesupavan) [reply generated/provider_dependencies.json:849]: We are using compat provider for imports to work with old core location. so here in this case msgraph uses timedelta, https://github.com/apache/airflow/pull/43571/files/a750af6fec2a04ba5a347303fb4397a4e496d56e#diff-af3b61e0f2e76a9841ce87162351100deee4bc736a90d4cb8715acd9fb61473cR23 

pre-commit generated this with common.compat as cross provider

**Comment by gopidesupavan** [comment]
Does email operator/email utils is required to move inside standard provider?

**Comment by gopidesupavan** [comment]
This pr getting more and more changes i feel and some changes require in selective checks to remove `OPERATORS` tests from core area. So i feel better to have split the triggers to separate pr #43608.

Will update this later.

**Comment by potiuk** [comment]
> This pr getting more and more changes i feel and some changes require in selective checks to remove `OPERATORS` tests from core area. So i feel better to have split the triggers to separate pr #43608.
> 
> Will update this later.

Makes sense

**Comment by eladkal** [comment]
Do we still need this PR or was it break into smaller ones that were already merged?

**Comment by gopidesupavan** [comment]
> Do we still need this PR or was it break into smaller ones that were already merged?

Don't think this is required, kept open to double check the final checks. I think we are good I will close this.

**Pull request closed without merging**